rotkehlchen/tests/data/exported_assets.json

```diff
@@ -1 +1,86 @@
-{ "version": "3", "assets": [ { "identifier": "eip155:1/erc20:0xFEEf77d3f69374f66429C91d732A244f074bdf74", "name": "Convex FXS", "symbol": "cvxFXS", "asset_type": "evm token", "started": null, "forked": null, "swapped_for": null, "cryptocompare": "", "coingecko": null, "address": "0xFEEf77d3f69374f66429C91d732A244f074bdf74", "evm_chain": "ethereum", "token_kind": "erc20", "decimals": 18, "protocol": null, "underlying_tokens": [ { "address": "0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0", "weight": "100", "token_kind": "erc20" } ] }, { "identifier": "b3a59173-7275-42e2-b3b1-4f5e4abd7878", "name": "MIMATIC", "symbol": "MAI", "asset_type": "other", "started": null, "forked": null, "swapped_for": null, "cryptocompare": "", "coingecko": "mimatic" }, { "identifier": "eip155:1/erc20:0xaC14864ce5A98aF3248Ffbf549441b04421247D3", "name": "Staked SDT", "symbol": "xSDT", "asset_type": "evm token", "started": null, "forked": null, "swapped_for": null, "cryptocompare": "", "coingecko": "stake-dao", "address": "0xaC14864ce5A98aF3248Ffbf549441b04421247D3", "evm_chain": "ethereum", "token_kind": "erc20", "decimals": 18, "protocol": null, "underlying_tokens": null }, { "identifier": "eip155:1/erc20:0xA0446D8804611944F1B527eCD37d7dcbE442caba", "name": "1INCH Token (Staked)", "symbol": "st1INCH", "asset_type": "evm token", "started": null, "forked": null, "swapped_for": null, "cryptocompare": "", "coingecko": null, "address": "0xA0446D8804611944F1B527eCD37d7dcbE442caba", "evm_chain": "ethereum", "token_kind": "erc20", "decimals": 18, "protocol": null, "underlying_tokens": [ { "address": "0x111111111117dC0aa78b770fA6A738034120C302", "weight": "100", "token_kind": "erc20" } ] } ] }
+{
+    "version": "3",
+    "assets": [
+        {
+            "identifier": "e7fa04fa-2473-49c1-898c-09332d8ee3e9",
+            "asset_type": "custom asset",
+            "name": "AAPL",
+            "custom_asset_type": "Stock",
```

Comment:
This file should be minified

Reply:
Would it make much sense? It is not a huge file and having it the way it is right now makes it more readable.